Source/bmalloc/libpas/src/libpas/pas_large_sharing_pool.c

```diff
@@ -531,11 +531,11 @@ static void splat_live_bytes(pas_large_sharing_node* node,
     bool did_overflow;
     switch (free_mode) {
     case pas_free:
-        did_overflow = __builtin_usubl_overflow(node->num_live_bytes, delta, &node->num_live_bytes);
+        did_overflow = __builtin_usubll_overflow((unsigned long long) node->num_live_bytes, delta, (unsigned long long *) &node->num_live_bytes);
```

Comment:
Let's have `PAS_TESTING_ASSERT(sizeof(size_t) == sizeof(unsigned long long))`. Otherwise, this is wrong.